packages/components/.npmignore

```diff
@@ -0,0 +1 @@
+.tsbuildinfo
```

Comment:
.npmignore? There isn't a yarn equivalent?

Reply:
Yarn will also read this file, and I don't think a `.yarnignore` file exists.